### PR TITLE
add support for host keys and single daemon container

### PR DESCRIPTION
**Pull request by tourunen**
- heat stack now includes optional host keys in cloud-init user data
- deprovisioning adapted to unregister runners with the new
  single daemon container approach

### CODE REVIEW

File: files/gitlab-runner-stack.yml [hunk 1]
```diff
@@ -34,8 +34,39 @@ parameters:
...
+    default: ''
+  runner_ssh_key_ecdsa_public:
+    description: >
+      Preconfigured ECDSA host public key
+    type: string
+    default: ''
 
 resources:
+  cloud_config_ssh_keys:
+    # this resource demonstrates multiple cloud-config resources
```
Comment (rlaurika) [inline files/gitlab-runner-stack.yml:60]: Does it? How? Not that familiar with cloud_config.

**Review by rlaurika — CHANGES_REQUESTED** [review]
The change looks good, but I'm a bit puzzled by one comment.

File: files/gitlab-runner-stack.yml (hunk 1 above)
Comment (tourunen) [reply files/gitlab-runner-stack.yml:60]: Ah, 'demonstrates' is a giveaway (copy paste + modifications from an example). Fixing.

**Comment by tourunen** [comment]
closing this in favor of #4

**Pull request closed without merging**